### PR TITLE
Can't enter independent camera mode through edit.js in HMD mode

### PR DESCRIPTION
**Pull request by SamGondelman**
Right now it's possible, using edit.js in an HMD, to enter Independent camera mode and have your view stuck to an object, which makes you sick very easily.  This makes it so you can't get in that state while in an HMD.

Combined with https://github.com/highfidelity/hifi/pull/8123, you also won't be able to get into that state by starting in Desktop mode.  If we decide not to merge that PR, I'll figure out another way of preventing that.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: cf44783550a906679324fa5f13847b617ec4dd3c available at: https://gist.github.com/20674da3320db2028bdd321b8633fb7e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.dmg

**Comment by huffman** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: cf44783550a906679324fa5f13847b617ec4dd3c available at: https://gist.github.com/20674da3320db2028bdd321b8633fb7e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: cf44783550a906679324fa5f13847b617ec4dd3c available at: https://gist.github.com/20674da3320db2028bdd321b8633fb7e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.dmg

File: scripts/system/libraries/entityCameraTool.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ CameraManager = function() {
     };
 
     that.enable = function() {
-        if (Camera.mode == "independent" || that.enabled) return;
+        if (Camera.mode == "independent" || that.enabled || HMD.active) return;
```
Comment (ZappoMan) [inline scripts/system/libraries/entityCameraTool.js]: coding standard: brace placement... should be:

```
 if (Camera.mode == "independent" || that.enabled || HMD.active) {
    return;
}
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: a0bb55e4c8579679b7832542bb6b67459fb3c109 available at: https://gist.github.com/20674da3320db2028bdd321b8633fb7e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.dmg

**Comment by SamGondelman** [comment]
erf I screwed that up, I'll put up another PR.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a358e3c2d9f91f1c2215ffacab9ee62fdfcfeff8 available at: https://gist.github.com/20674da3320db2028bdd321b8633fb7e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8124.dmg

**Pull request closed without merging**